experiments/dsl/src/main/scala/org/apache/gearpump/streaming/dsl/StreamApp.scala

```diff
@@ -100,15 +100,13 @@ class TimeReplayableProducer[T:ClassTag](timeReplayableSource: TimeReplayableSou
 }
 
 class KafkaProducer[T:ClassTag](kafkaConfig: KafkaConfig, converter: Message => T) extends Traverse[T] {
```

Comment:
better move the KafkaProducer to kafka package instead of DSL package. Otherwise DSL package compilation will need kafka package.